services/movie_session.py

```diff
@@ -41,4 +42,13 @@
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    get_user_model().objects.get(id=session_id).delete()
```

Comment:
It seems like there is a mistake here. You are trying to delete a user model instance, but it should be a movie session instance. The function name `delete_movie_session_by_id` also suggests that a movie session should be deleted. Please revise this part.